frontend/src/Editor/QueryManager/Components/Preview.jsx

```diff
@@ -1,18 +1,26 @@
 import React, { useEffect, useLayoutEffect, useRef, useState } from 'react';
 import { JSONTree } from 'react-json-tree';
-import { Tab, ListGroup, Row, Col } from 'react-bootstrap';
-import { usePreviewLoading, usePreviewData, useQueryPanelActions } from '@/_stores/queryPanelStore';
+import { Tab, ListGroup, Row, Col, Table } from 'react-bootstrap';
+import {
+  usePreviewLoading,
+  usePreviewData,
+  useQueryPanelActions,
+  useSelectedDataSource,
+} from '@/_stores/queryPanelStore';
 import { getTheme, tabs } from '../constants';
 import RemoveRectangle from '@/_ui/Icon/solidIcons/RemoveRectangle';
 import { ButtonSolid } from '@/_ui/AppButton/AppButton';
 
 const Preview = ({ darkMode }) => {
   const [key, setKey] = useState('raw');
+  const [columns, setColumns] = useState(null);
   const [isJson, setIsJson] = useState(false);
   const [theme, setTheme] = useState(() => getTheme(darkMode));
   const queryPreviewData = usePreviewData();
   const previewLoading = usePreviewLoading();
   const { setPreviewData } = useQueryPanelActions();
+  const { kind } = useSelectedDataSource();
+  //console.log("jhgfcdxzcfghjmhgdxsdfghjmkmjhgfcdx", kind)
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/ToolJet/ToolJet/pull/10065#issuecomment-2163104325>#17d927</a><span> - 06/12/2024, 02:07 pm</span>
<br>

<div><small>🟠</small> Medium importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> There is a commented-out console.log statement left in the code. This can lead to unnecessary clutter and potential performance issues if accidentally uncommented.
</div>
<div id="fix">
<b>Fix:</b> Remove the commented-out console.log statement to keep the code clean and maintainable.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div>
<pre><code>
 @@ -23 +23 @@
 -  //console.log("jhgfcdxzcfghjmhgdxsdfghjmkmjhgfcdx", kind)
</code></pre>
</div>
</details>
</div>
<br>


</div>